src/CineFiles.jl

```diff
@@ -59,9 +258,13 @@ function CineHeader(fname)
     open(fname) do f
         read(f, UInt16) == UInt(18755) || error(basename(fname), " is not a .cine file")
         seek(f, 0)
-        cine = readstruct(f, CineFileHeader)
+        cine = read(f, CineFileHeader)
         seek(f, cine.OffImageHeader)
-        bitmap = readstruct(f, BitmapInfoHeader)
+        bitmap = read(f, BitmapInfoHeader)
+
+        seek(f, 0)
```

Comment:
`seek` specifies a position, not an offset, so only one `seek` is needed

Reply:
👍